lib/pod-names.js

```diff
@@ -0,0 +1,32 @@
+var Plugin = require('broccoli-plugin');
+var walkSync = require('walk-sync');
+var fs = require('fs');
+var path = require('path');
+var componentNames = require('./component-names.js');
+
+module.exports = PodNames;
+
+PodNames.prototype = Object.create(Plugin.prototype);
+PodNames.prototype.constructor = PodNames;
+function PodNames(inputNode, options) {
+  options = options || {};
+  Plugin.call(this, [inputNode], {
```

Comment:
@stefanpenner a little curious as to how to do this with a filter. I'm wanting to take a set of files, and turn part of their paths into a json blob that can get exported to be imported after build into the initializer. If the `processString` method outputted a key value pair, could you then use the `postProcess` method to join all of those into a single output?